src/vs/workbench/contrib/terminal/common/terminal.ts

```diff
@@ -748,9 +748,9 @@ export const terminalContributionsDescriptor: IExtensionPointDescriptor<ITermina
 		description: nls.localize('vscode.extension.contributes.terminal', 'Contributes terminal functionality.'),
 		type: 'object',
 		properties: {
-			quickFixes: {
```

Comment:
I think this is still under `terminal.terminalQuickFixes` because of `extensionPoint: 'terminal'` above